exporter/kafkaexporter/kafka_exporter.go

```diff
@@ -19,7 +19,11 @@ import (
 	"fmt"
 
 	"github.com/Shopify/sarama"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
```

Comment:
I'm somewhat wary of pulling OpenCensus dependencies into more components when we know we need to move away from it.


@bogdandrutu @tigrannajaryan are there alternatives to this?  I think what is being tracked here is slightly different from what `obsreport` produces.

Reply:
Hi, does it means we need to wait for the other alternatives to be available if we want to add more custom metrics to the obsreport metrics? Or is it acceptable to include it for now?

Reply:
I'd like to hear from the maintainers what their thoughts on the matter are.  I'm not sure what the right thing is here.

Reply:
Understood.
Gentle ping @bogdandrutu @tigrannajaryan please take a look of this when you have time